modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -21,23 +21,20 @@ These are suitable for a quick test installation, such as a server with one clie
 If you want to use a production environment, review the requirements listed in xref:installation-and-upgrade:hardware-requirements.adoc[].
 ====
 
-In the following table replace {sles-version}-{sp-base} with the actual product version string.
-Find the {minimalsles} image at https://download.suse.com/.
-
 [cols="1,1", options="header"]
 |===
-2+<| KVM Settings
+2+<| Virtual Machine Settings overview
 | Installation Method | Import Existing Disk Image
-| OS:                 | Linux
-| Version:            | SLES{sles-version}-{sp-base}-{minimal}-VM.x86_64-kvm-and-xen-GM.qcow2
+| OS:                 | SUSE Linux Enterprise 15 SP4
 | Memory:             | 16 GB
 | CPU's:              | 4
-| Storage Format:     | _.qcow2_ 24 GB (Default) Minimal Root Partition
 | Virtual Disks:      |
+| VirtIO Disk 1       | SUSE-Manager-Server.x86_64-4.3.10-KVM.qcow2
 | VirtIO Disk 2       | 101 GB for [path]``/var/spacewalk``
 | VirtIO Disk 3       | 50 GB for [path]``/var/lib/pgsql``
 | VirtIO Disk 4       | 4 GB for swap
```

Comment:
With adding the disk as swap here there is the problem down in the `suma-storage` command that this swap partition is not being used, we haven't mounted it properly. Is there a way to add it in the `suma-storage`? Or another way to use it in the VM? Otherwise the SUMA configuration creates a 2GB swap from the root partition on its own.

Reply:
Good point. I missed the swap entry. Size of the root is set to 15GB, when image is deployed, there are 11GB available. So taking 2GB from it is kind of big chunk of available storage.

`suma-storage` is maintained by cloud team at https://github.com/SUSE-Enceladus/susemanager-cloud-setup so it will need to be discussed with them or create PR for it there.

Reply:
Ok great then if we can create a new PR or a card for that, it would be a separate issue and we can resolve this. I will resolve the conversation here to be able to merge this current PR, it should go out as soon as possible with the next update.